src/DependencyInjection/MonologExtension.php

```diff
@@ -720,6 +720,7 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
                 break;
 
             case 'sentry':
+                @trigger_error('Since symfony/monolog-bundle 3.11: The "sentry" handler type is deprecated, use the "sentry/sentry-symfony" and a "service" handler instead.');
```

Comment:
Any reason why [`trigger_deprecation`()](https://github.com/symfony/deprecation-contracts/tree/main) is not used?